trikotFoundation/src/commonMain/kotlin/com/mirego/trikot/foundation/concurrent/dispatchQueue/SequentialDispatchQueue.kt

```diff
@@ -17,26 +17,18 @@ open class SequentialDispatchQueue(override val dispatchQueue: TrikotDispatchQue
     override fun isSerial() = true
 
     override fun dispatch(block: DispatchBlock) {
-        if (isSynchronous && currentDispatch.compareAndSet(NoDispatchBlock, SyncDispatchBlock)) {
```

Comment:
Shouldn't we remove `isSynchronous` if we don't use it anymore?

Reply:
I agree, but I think it would be a breaking change since this  could be a breaking change since this variable is open.